src/app/utils/field-validation/validators/childrenValidator.ts

```diff
@@ -24,7 +24,9 @@ const childrenAnswerValidator: ChildrenValidator = (response) => {
   const { answerArray } = response
 
   return answerArray.length === 0
-    ? left(`ChildrenValidator:\t Answer is empty array`)
+    ? left(
+        `ChildrenValidator (childrenAnswerValidator):\t Answer is empty array`,
```

Comment:
Not very familiar with this part, but is the addition of ` (childrenAnswerValidator)` intended?

Reply:
I added the function name in the brackets so it's clearer where the error message is thrown from, cos two of the error messages in `childrenValidator` were the same. It's not necessary for this particular one though (since the error message is unique), but left it in there anyway to keep the error messages consistent

Reply:
I see. It was asking in terms of all the validator errors thrown. Thought that it was some debug text, but your reasoning makes sense too.